inferno-framework/davinci-dtr-test-kit
# FI-3435: Documentation for DTR Light EHR release

# Summary
This PR updates the documentation of the DTR Light EHR test suite and general DTR test kit and home page documentation in preparation for the initial publication of the Light EHR suite.

## Code Changes
- I tried to organize the RESTful capability profiles in a better way by following the structure of the USCore v3.1.1 fhir api test group. I created a group.rb file for each of the profiles so that I could include detailed descriptions for each without cluttering the `dtr_light_ehr_suite.rb` file.
- `docs/dtr_light_ehr_suite_description_v201.md` 
    - Added DTR QuestionnaireResponse and CDex Task explanations to limitations section (is this necessary? is this the spot for it?)

# Testing Guidance
- Read through updates to documentation and make sure there are no typos/errors.
- Make sure details are relevant and not redundant.
- Let me know if anything is unclear/missing.
- Run the test kit as if you were a branch new user and make sure the instructions make sense.

## Questions
1. Should I update the preset to use the public instance of the inferno reference server instead of localhost now that that patient has been merged? 
2. What should I name test groups 2 and 3? One is for US Core API and the other is for the DTR Light EHR profile API.
